com.unity.ml-agents/Runtime/Academy.cs

```diff
@@ -58,13 +58,13 @@ public class Academy : IDisposable
         /// on each side, although we may allow some flexibility in the future.
         /// This should be incremented whenever a change is made to the communication protocol.
         /// </summary>
-        const string k_ApiVersion = "0.15.0";
+        const string k_ApiVersion = "0.16.0.dev0";
 
         /// <summary>
         /// Unity package version of com.unity.ml-agents.
         /// This must match the version string in package.json and is checked in a unit test.
         /// </summary>
-        internal const string k_PackageVersion = "0.14.1-preview";
+        internal const string k_PackageVersion = "0.16.0-preview";
```

Comment:
@vincentpierre what convention are we following for package versioning?

Reply:
Hmm, something seems off here. I do not know our convention but maybe I should do `0.15.0-preview` and have the same on release and master (like it was the case for 0.14.1)

Reply:
Yeah, I think you need to 0.15.0-preview or else the changelog checker complains (it might complain anyway). We didn't quite get the process figured out for this yet.